lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -25,6 +25,11 @@ $nx-global-sidebar-focus-border: 1px solid $nx-focus-border-color;
 $nx-global-sidebar-separator-color: $nx-grey-900;
 
 .nx-global-sidebar {
+  // Override default dark grey & black text colours for white in the sidebar
+  $nx-heading-color: $nx-white;
+  $nx-text-color: $nx-white;
+  $nx-link-color: $nx-white;
```

Comment:
None of these do anything.  This overrides these values in a lexical sense not in a CSS cascade sense.  In other words, only usages of `$nx-heading-color`, `$nx-text-color`, and `$nx-link-color` that are present within this scss block would be affected by these declarations – and there aren't any.

Reply:
> None of these do anything. This overrides these values in a lexical sense not in a CSS cascade sense. In other words, only usages of `$nx-heading-color`, `$nx-text-color`, and `$nx-link-color` that are present within this scss block would be affected by these declarations – and there aren't any.

Oh yeah those should have been removed a while ago: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/215b76ae1d8d651406416bab2f53e5f43329836f